src/main/java/io/github/thebusybiscuit/slimefun4/utils/RechargeableHelper.java

```diff
@@ -8,32 +8,38 @@
 
 import javax.annotation.Nonnull;
 
+import io.github.thebusybiscuit.slimefun4.core.attributes.Rechargeable;
+import org.apache.commons.lang.Validate;
 import org.bukkit.NamespacedKey;
 import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.persistence.PersistentDataType;
 
 import io.github.thebusybiscuit.cscorelib2.chat.ChatColors;
 import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
-import io.github.thebusybiscuit.slimefun4.utils.PatternUtils;
 import net.md_5.bungee.api.ChatColor;
 
 /**
  * This is just a simple helper class to provide static methods to the {@link Rechargeable}
  * interface.
- * 
+ *
  * @author TheBusyBiscuit
- * 
+ * @author WalshyDev
+ *
  * @see Rechargeable
  *
  */
-final class RechargeableHelper {
+public final class RechargeableHelper {
```

Comment:
We could rename this from RechargeableHelper to `ChargeUtils` perhaps? Or have a better suggestion?